### ISSUE TITLE
RFE: Ability to skip embedding container payload

### ISSUE THREAD

**Issue by lzap** [osbuild/bootc-image-builder#393]
I find useful to be able to generate ISO that does not actually carry the container repository payload, just the kickstart with the upstream URL where Anaconda can install from. I actually made a simple shell script that converts an official "netboot" Fedora or RHEL installation ISO into such artifact the other day:

https://github.com/lzap/bootc-mkiso

But this is dirty, I am wondering if there could be a bib switch to disable container payload embeding. When disabled, the upstream URL would be used instead.

**Comment by achilleas-k** [comment]
I don't know if this is a useful feature to have in BIB.  This is quite simple to accomplish with [mkksiso](https://weldr.io/lorax/mkksiso.html) from Lorax.  All you need is an Anaconda ISO and a kickstart file.

Meanwhile, doing it in BIB would require:
- Pulling the container to inspect it, so that BIB can determine what kind of ISO to make (distro version, repository URLs).
- Downloading RPMs from the repositories defined in the container.
- Throwing away the container because it wont be included in the ISO anyway.
- Building the ISO with the necessary kickstart file.

That first step is where things get weird and wasteful.  I'd much rather we supported this in osbuild-composer, where we could have a "net-install" ISO for different types of payloads, including bootc containers.

**Comment by achilleas-k** [comment]
> I want this to be in the hosted image-builder service so it can be leveraged on ConsoleDot. You mention osbuild-composer, so I guess I just asked at the wrong place. Can you move the RFE into that project for me?

Yeah, getting this as a feature in osbuild-composer would be the way to go to bring it into the IB service in console-dot.  Building pre-configured PXE ISOs is a nice idea for IB and osbuild-composer, I agree.
